packages/frontend2/src/app/(new)/(other)/bridges/risk/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { BridgesMvpWarning } from '~/app/_components/bridges-mvp-warning'
+import { SimplePageHeader } from '~/app/_components/simple-page-header'
+import { getBridgeRiskEntries } from '~/server/features/bridges/get-bridge-risk-entries'
+import { getDefaultMetadata } from '~/utils/get-default-metadata'
+import { BridgesFilterContextProvider } from '../_components/bridges-filter-context'
+import { BridgesRiskTables } from './_components/table/bridges-risks-tables'
+
+export const metadata = getDefaultMetadata({
+  openGraph: {
+    url: '/bridges/summary',
```

Comment:
This should be `/bridges/risk`

Reply:
https://github.com/l2beat/l2beat/pull/4697/commits/9f8d7d9f05ca0efe749006483a3985590bdb745d